pkg/instrumentation/sdk.go

```diff
@@ -502,6 +503,26 @@ func (i *sdkInjector) addParentResourceLabels(ctx context.Context, uid bool, ns
 			if uid {
 				resources[semconv.K8SJobUIDKey] = string(owner.UID)
 			}
+
```

Comment:
Can we also add a unit test, while we're at it? LGTM otherwise.

Reply:
sure, I'll take a look and get back to you soon.

Reply:
done, pls give it another review :-)